docs/dev/building-on-zksync/hello-world.md

```diff
@@ -412,16 +375,16 @@ const GREETER_CONTRACT_ADDRESS = "0x...";
 const GREETER_CONTRACT_ABI = require("./abi.json");
 ```
 
-### Working with provider
+### Working with a Web3 Provider
 
-1. Go to the `initializeProviderAndSigner` method in `./src/App.vue`. This method is called after the connection to Metamask is successful.
+1. Go to the `initializeProviderAndSigner` function in `./src/App.vue`. This function is called after the connection to Metamask is successful.
 
-In this method we should:
+In this function we should:
 
 - Initialize a `Web3Provider` and a `Signer` to interact with zkSync.
 - Initialize the `Contract` object to interact with the `Greeter` contract we just deployed.
 
-2. Import the necessary dependencies:
+2. Import the necessary dependencies under the imports from before:
 
```

Comment:
```suggestion
2. Import the necessary dependencies under the imports from before:

```

Leaving the previous text "Import the necessary dependencies:", seems clearer, I think?

Reply:
Not accepting this one because it is not clear to a newbie where to put these new imports.